code/datums/elements/atmos_sensitive.dm

```diff
@@ -33,8 +32,7 @@
 	SIGNAL_HANDLER
 	var/atom/atom_source = source
 	if(isopenturf(atom_source.loc))
```

Comment:
this ifcheck is redundant

Reply:
Is it though? what if it somehow ends up inside a wall LOL like someone builds a wall on top of a scrubber? that's what this is for right?

Reply:
atmos_conditions_changed will handle it for us

Reply:
Yeeeeeeeet